packages/lib/server/checkUsername.ts

```diff
@@ -3,4 +3,6 @@ import { IS_PREMIUM_USERNAME_ENABLED } from "@calcom/lib/constants";
 import { checkRegularUsername } from "./checkRegularUsername";
 import { usernameCheck as checkPremiumUsername } from "./username";
 
+// FIXME: Why don't we use checkPremiumUsername from "@calcom/features/ee/common/lib/checkPremiumUsername" here?
```

Comment:
Discussed with @sean-brydon. We should replace lib/checkPremiumUsername  with `usernameCheck` which completely removes dependency on website's `username` endpoint from the app.